Cargo.toml

```
@@ -13,6 +13,8 @@ keywords = ["machine-learning", "linfa", "ai", "ml"]
 categories = ["algorithms", "mathematics", "science"]
 
 [dependencies]
+
+linfa-supervised = {path = "linfa-supervised"}
```

You need to add a version number, of publishing the `linfa` crate to crates.io will fail :wink:
```suggestion
linfa-supervised = { path = "linfa-supervised", version = "0.1" }
```